Add feature flag called events clarification for billing (#8721)
PostHog/posthog.com: contents/docs/feature-flags/common-questions.mdx

> **Note:** Do not use local evaluation in an edge / lambda environment, as this initializes a PostHog instance on every call, which can raise your bill drastically. It's best to use regular flag evaluation instead.

## I have very few `$feature_flag_called` events when I look at a flag, but my bill is still very high, why is this?

> The usage you see for feature flags is for auditing and debugging how your flags are being used (and as a result, can be easily switched off). Further, they're not sent every time, just when the flag value changes for a person.

Reviewer: Could we be more explicit about what triggers a $feature_flag_called event?

> The usage you see for feature flags is for auditing and debugging how your flags are being used (and as a result, can be easily switched off). Further, they're not sent every time, just when the flag value changes for a person.

Author: ```suggestion
The usage you see for feature flags is for auditing and debugging how your flags are being used (and as a result, can be easily switched off). By default, these events are sent whenever you call `getFeatureFlag()` or `isFeatureEnabled()`. Further, they're not sent every time, just when the flag value changes for a given person. 
```

> ## I have very few `$feature_flag_called` events when I look at a flag, but my bill is still very high, why is this?
> 
> The usage you see for feature flags is for auditing and debugging how your flags are being used (and as a result, can be easily switched off). By default, these events are sent whenever you call `getFeatureFlag()` or `isFeatureEnabled()`. Further, they're not sent every time, just when the flag value changes for a given person. 
> 
> This is different from requests made to our servers to compute flags, which is what feature flag billable requests are. Thus, generally, there will be no relation between the no. of feature flag called events and the number you see in billing.
> 

Reviewer 2: ```suggestion
## I have very few `$feature_flag_called` events when I look at a flag, but my bill is still very high, why is this?

$feature_flag_called` events are captured whenever you call `getFeatureFlag()` or `isFeatureEnabled()`. Also, they're not sent every time and are only sent when the flag value changes for a given person. 

This is different from requests made to our servers to compute flags, which is what feature flag billable requests are. Thus, generally, there will be no relation between the number of `$feature_flag_called` events and the usage you see in your billing.
```

Reviewer 2: @neilkakkar probably a follow up question to this will be "How do I reduce my feature flag billing". Is there a good answer to this?

Author: Yep, the question above should answer that
